charts/ethereum/values.yaml

```diff
@@ -0,0 +1,286 @@
+# to get to the requested state.
+# Specifying a lower number of slots as the frequency increases the disk space usage.
+dataStorageArchiveFrequency: "8192"
+
+
+# Extra flags to pass to the node
+extraFlags:
+  executionClient: []
+  consensusClient: []
+
+imageRepository:
+  geth: ethereum/client-go
+  nethermind: nethermind/nethermind
+  erigon: thorax/erigon
+  prysm: gcr.io/prysmaticlabs/prysm/beacon-chain
+  prysmGnosis: ghcr.io/gnosischain/gbc-prysm-beacon-chain
+  lighthouse: sigp/lighthouse
+  teku: consensys/teku
+imageTag:
+  geth: v1.10.21
+  nethermind: 1.13.4
+  erigon: v2022.07.03
+  prysm: v2.1.4-rc.0
+  prysmGnosis: v2.1.2-gbc
+  lighthouse: v2.5.0
+  teku: 22.7.0
+imagePullPolicy: IfNotPresent
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Init image is used to chown data volume, initialise genesis, etc.
+initImage:
+  repository: "busybox"
+  tag: "1.35"
+  pullPolicy: IfNotPresent
+
+# Sidecar image is used to perform Liveness/Readiness probes.
+sidecar:
+  repository: "europe-west4-docker.pkg.dev/stakewiselabs/public/ethnode-sidecar"
+  tag: "v1.0.3"
+  pullPolicy: IfNotPresent
+  bindAddr: "0.0.0.0"
+  bindPort: 3000
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations:
+  geth: {}
+  nethermind: {}
+  erigon: {}
+  prysm: {}
+  lighthouse: {}
+  teku: {}
+
+securityContext:
+  geth:
+    fsGroup: 1001
+    runAsUser: 1001
+  nethermind:
+    fsGroup: 1000
+    runAsUser: 1000
+  erigon:
+    fsGroup: 1001
+    runAsUser: 1001
+  prysm:
+    fsGroup: 1001
+    runAsUser: 1001
+  lighthouse:
+    fsGroup: 1001
+    runAsUser: 1001
+  teku:
+    fsGroup: 1000
+    runAsUser: 1000
+
+service:
+  type: ClusterIP
+
+# When p2pNodePort is enabled, your P2P port will be exposed via service type NodePort.
+# This will generate a service for each replica, with a port binding via NodePort.
+# This is useful if you want to expose and announce your node to the Internet.
+p2pNodePort:
+  # Expose P2P port via NodePort
+  enabled: false
+  annotations: {}
+  # Options: NodePort, LoadBalancer
+  type: NodePort
+  # The ports allocation will start from this value
+  startAt:
+    executionClient: 31100
+    consensusClient: 31200
+  # Overwrite a port for specific replicas
+  replicaToNodePort:
+    executionClient: {}
+      #  "0": 32345
+      #  "3": 32348
+    consensusClient: {}
+      #  "0": 32345
+      #  "3": 32348
+  initContainer:
+    image:
+      repository: bitnami/kubectl
+      tag: "1.24"
+      pullPolicy: IfNotPresent
+
+resources:
+  executionClient: {}
+  consensusClient: {}
+
+nodeSelector:
+  executionClient: {}
+  consensusClient: {}
+
+tolerations:
+  executionClient: []
+  consensusClient: []
+
+affinity:
```

Comment:
Should we collocate consensus and execution clients on the same node as they always work in pairs?

Reply:
No, this is not very necessary, also some operators have already expressed a desire to run them on separate machines